typescript/web/src/components/workspaces/update-plan-banner.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { createCommonDecorator, storybookTitle } from "../../utils/stories";
+import { WORKSPACE_DATA } from "../../utils/fixtures";
+import { UpdatePlanBanner } from "./update-plan-banner";
+
+export default {
+  title: storybookTitle("Workspaces", "UpdatePlanBanner"),
+  decorators: [
+    createCommonDecorator({
+      auth: { withWorkspaces: true },
+      apollo: true,
+      router: { query: { workspaceSlug: WORKSPACE_DATA.slug } },
+    }),
+  ],
+};
+
+export const Open = () => <UpdatePlanBanner />;
```

Comment:
It's kind of an hidden Storybook feature but if you name your single story with the same name as the component it exposes, you will remove the useless extra-level in the left-sidebar 😉 

Also, we should give the whole component as last argument to `storybookTitle` instead of just it's name so that it not more portable.

```suggestion
import { UpdatePlanBanner as UpdatePlanBannerComponent } from "./update-plan-banner";

export default {
  title: storybookTitle("Workspaces", UpdatePlanBannerComponent),
  component: UpdatePlanBannerComponent,
  decorators: [
    createCommonDecorator({
      auth: { withWorkspaces: true },
      apollo: true,
      router: { query: { workspaceSlug: WORKSPACE_DATA.slug } },
    }),
  ],
};

export const UpdatePlanBanner = () => <UpdatePlanBannerComponent />;
```